### PR TITLE
[homebrew] Adapt to new JSON API of formulae.brew.sh

### PR DESCRIPTION
**Pull request by koraktor**
[formulae.brew.sh](formulae.brew.sh) has switched to a new JSON API which offers the complete formula data.

### CODE REVIEW

**Comment by RedSparr0w** [comment]
Thanks for submitting this PR!
Could you please update the title to include `[homebrew]` to allow the tests to run.
eg: `[homebrew] Adapt to new JSON API of formulae.brew.sh`

File: server.js [hunk 1]
```diff
@@ -6398,7 +6398,7 @@ camp.route(/^\/homebrew\/v\/([^/]+)\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
   var pkg = match[1];  // eg. cake
   var format = match[2];
-  var apiUrl = 'http://formulae.brew.sh/formula/' + pkg + '/version';
+  var apiUrl = 'http://formulae.brew.sh/formula/api/' + pkg + '.json';
```
[the remark this answers is not in the document]
Comment (koraktor) [reply server.js]: 👍 Thanks!

**Comment by PyvesB** [comment]
Looks good to me as well, merged. Thank you for your contribution @koraktor !

**Comment by koraktor** [comment]
@RedSparr0w @PyvesB 

Is this already in production? The badges still show as `not found`.

**Comment by RedSparr0w** [comment]
Not yet unfortunately, you can check to see the commits awaiting deployment to the live server [here](https://github.com/badges/shields/compare/gh-pages...master)

Quoting @paulmelnikow:

> Deploys usually happen every 1–3 weeks. Thaddée [@espadrine], who has limited time on this project, is the only sysadmin. He's working on giving me access to deploy and logs, but doing so is complicated because the hosting account (and maybe the servers too) are shared with other services he runs.

It's been a couple month's now since the last one (Mar 25), so hopefully we should get a deployment soon.
🤞

**Comment by gep13** [comment]
That would be great! Looking forward to using these home brew badges again.

**Pull request merged**